react/features/room-lock/middleware.js

```diff
@@ -38,6 +40,8 @@ MiddlewareRegistry.register(store => next => action => {
             if (error.recoverable) {
                 store.dispatch(_openPasswordRequiredPrompt(conference));
             }
+        } else {
+            store.dispatch(hideDialog(RoomLockPrompt));
```

Comment:
:philosoraptor: `RoomLockPrompt` and not `PasswordRequiredPrompt`?!

Reply:
Hum, actually it should be both. You can leave this one out and I'll take a look as soon as I'm back!